src/SFA.DAS.ApplyService.Data/ApplyRepository.cs

```diff
@@ -82,7 +82,7 @@ public async Task<List<GatewayPageAnswerSummary>> GetGatewayPageAnswers(Guid app
         {
             using (var connection = new SqlConnection(_config.SqlConnectionString))
             {
-                return (await connection.QueryAsync<Domain.Entities.GatewayPageAnswerSummary>(@"SELECT applicationId,pageid,status from GatewayAnswer
+                return (await connection.QueryAsync<Domain.Entities.GatewayPageAnswerSummary>(@"SELECT applicationId,pageid,status, Comments from GatewayAnswer
```

Comment:
Please correct the Capitalisation of the columns

ApplicationId, PageId, Status, Comments

Reply:
Done.